tests/system/knowledge_base/qdrant/test_qdrant_knowledge_base.py

```diff
@@ -0,0 +1,323 @@
+import random
+from copy import copy
+from pathlib import Path
+
+import pytest
+from dotenv import load_dotenv
+from qdrant_client import QdrantClient
+from canopy.knowledge_base.chunker.base import Chunker
+
+from canopy.knowledge_base.knowledge_base import KnowledgeBase
+from canopy.knowledge_base.qdrant.qdrant_knowledge_base import (
+    DENSE_VECTOR_NAME,
+    QdrantConverter,
+    QdrantKnowledgeBase,
+)
+from canopy.knowledge_base.qdrant.qdrant_knowledge_base import COLLECTION_NAME_PREFIX
+from canopy.knowledge_base.models import DocumentWithScore
+from canopy.knowledge_base.record_encoder.base import RecordEncoder
+from canopy.knowledge_base.reranker.reranker import Reranker
+from canopy.models.data_models import Query
+
+from qdrant_client.qdrant_remote import QdrantRemote
+from canopy_cli.cli import _load_kb_config
+from tests.system.knowledge_base.qdrant.common import (
+    assert_chunks_in_collection,
+    assert_ids_in_collection,
+    assert_ids_not_in_collection,
+    assert_num_points_in_collection,
+    total_vectors_in_collection,
+)
+from tests.unit.stubs.stub_chunker import StubChunker
+from tests.unit.stubs.stub_dense_encoder import StubDenseEncoder
+from tests.unit.stubs.stub_record_encoder import StubRecordEncoder
+
+load_dotenv()
```

Comment:
Not required. We have `pytest-dotenv` installed by default now

Reply:
https://github.com/pinecone-io/canopy/pull/244/commits/289577501bf0b1263172c3e22d8f3fb16ce2ee5f